components/calendar/src/hebrew.rs

```diff
@@ -133,7 +154,8 @@ impl Calendar for Hebrew {
                 "M04" => 4,
                 "M05" => 5,
                 "M05L" => 6,
-                "M06" => 7,
+                // M06L is the formatting era code used for Adar II
+                "M06" | "M06L" => 7,
```

Comment:
Not convinced we should accept `"M06L"` here but I guess it's harmless

Reply:
not sure either :)